cloud-operator/Chart.yaml

```diff
@@ -3,6 +3,5 @@
 apiVersion: v1
 name: cloud-operator
 description: A Helm chart for deploying the Illumio cloud-operator application
-version: 0.0.1
-appVersion: "0.0.1"
+version: "1.0.0"
```

Comment:
Version here and version in values.yaml should mirror what the version is tagged in GHA from the build